packages/smooth_app/lib/pages/product/common/product_query_page.dart

```diff
@@ -203,10 +202,11 @@ class _ProductQueryPageState extends State<ProductQueryPage>
           actions: _getAppBarButtons(),
         ),
         body: RefreshIndicator(
-          onRefresh: () => refreshList(),
+          /// To allow refresh even when not the whole page is filled
```

Comment:
A bit too high this time: it's the `physics` parameter that is connected to this comment.
And it should be a `//` comment, not a `///` (here we don't explain our classes or a cunning method).